site/docs/workflows/spell_check.md

```diff
@@ -70,3 +70,46 @@ jobs:
       modified_files_only: false
       working_directory: examples/my_project
 ```
+
+## CSpell File Example
+
+More information can be found in [cspell docs](https://cspell.org/configuration/).
```

Comment:
Maybe we should also explain that this is like the "Very Good" Cspell that we recommend. Maybe even with a link to the very good dictionaries? 🤔

Reply:
@wolfenrain I took a different approach. With the example I tried to showcase different cspell parameters that can be set, such as the dictionaries (local and remote), allowed and not allowed words...

I like your suggestion about using an example that would be our recommendation. In such case, we leave out some parameters that can be found in cspell docs, but I think that is fine as long as we keep that sentence about finding more info in their docs.
wdyt?

Reply:
Ah gotcha, yea that sounds great as well to just have some more parameters. I do think we should refer to our dictionary repo as well tho